login_V1

```diff
@@ -1,37 +1,32 @@
+
 <?php
-	ob_start();
-	session_start();
-		$error = NULL;
-		if($_SERVER["REQUEST_METHOD"] == "POST") {
-			// connect.php (tells where to connect servername, dbaseName, username, password)
-			require "Database_mysqli.php";
-			// username and passwrod sent from form 
-			$myusername = mysqli_real_escape_string($conn, $_POST['username']);
-			$mypassword = mysqli_real_escape_string($conn, $_POST['password']);
-			
-			$query = "SELECT Username FROM Users WHERE Username = '$myusername' and Password = '$mypassword'";
-			
-			$result = mysqli_query($conn, $query);
-			$row = mysqli_fetch_array($result, MYSQLI_ASSOC);
+	$error = NULL;
+	if($_SERVER["REQUEST_METHOD"] == "POST") {
```

Comment:
I have replaced all of this redundant code with a more simple code that requests for a POST to have occurred in order to run.